docs/modules/ROOT/partials/nav-explanation.adoc

```diff
@@ -1 +1,4 @@
 * xref:k8up:ROOT:explanations/architecture.adoc[Architecture]
+* xref:k8up:ROOT:explanations/missing-docs.adoc[Missing Documentation]
```

Comment:
@tobru this page lists lots of stuff as missing. I figured I'll add a nav link to it. Was it intended to not be linked? 
Do we add documentation for those topics or after v1.0.0?

Reply:
I intentionally didn't link the page as I didn't see any value in it. It was a way to write it down.
Now that I think of it again it might make sense to link it as this gives the reader of the docs a way to see what else is in K8up but isn't documented yet.

>  Do we add documentation for those topics or after v1.0.0?
We add them as we go, there is plan _when_ to add it.

Reply:
> We add them as we go, there is plan when to add it.

I believe there's a _no_ missing?

Reply:
I have added a sentence in the page itself that says there are invisible features. Is that enough?